frontend/src/features/multisig/multisigSlice.js

```diff
@@ -185,6 +215,43 @@ export const signTx = createAsyncThunk(
   }
 );
 
+export const verifyAccount = createAsyncThunk(
+  "multisig/verifyAccount",
+  async (data, { rejectWithValue, dispatch }) => {
+    try {
+      const token = await window.wallet.signArbitrary(
+        data.chainID,
+        data.address,
+        VERIFICATION_MESSAGE
+      );
+      try {
+        const response = await multisigService.verifyUser({
+          address: data.address,
+          signature: token.signature,
+          salt: 10,
+          pubKey: JSON.stringify(token.pub_key),
+        });
+        return {
+          response: response,
```

Comment:
If both key and value names are the same, simply returning {response, token} is sufficient without defining key and object.